ui/src/pages/OBProxy/New/index.tsx

```diff
@@ -25,19 +47,22 @@ export default function New() {
             defaultMessage: '取消',
           })}
         </Button>,
-        <Button>
+        <Button onClick={() => form.submit()} type="primary">
           {intl.formatMessage({
             id: 'src.pages.OBProxy.New.E2247569',
             defaultMessage: '提交',
           })}
         </Button>,
       ]}
     >
-      <Form form={form}>
+      <Form onFinish={submit} form={form}>
         <Row gutter={[16, 16]}>
```

Comment:
If there is only one column in the `Row`, it seems not to be necessary to use `Row` component.

Reply:
It seems that using `Space` is a more reasonable choice.